lib/beaker-answers/pe_conf.rb

```diff
@@ -82,6 +82,19 @@ def _generate_1_0_data
         pe_conf["#{ns}::puppetdb_host"] = puppetdb.hostname
       end
 
+      if the_host_with_role('pe_postgres', raise_error=false)
+        pe_conf["#{ns}::database_host"] = the_host_with_role('pe_postgres', raise_error=false).hostname
+        if options[:pe_postgresql_options]
+          if options[:pe_postgresql_options][:security] == 'cert'
+            postgres_cert_answers(pe_conf, "1.0")
```

Comment:
![singlequotes](https://user-images.githubusercontent.com/759848/33956486-89c2ed2c-dff3-11e7-8c96-81c57506590f.jpg)

Reply:
This repo should really have a linter.

Reply:
Nevermind, I'll add a linter and auto-fix all this.